crates/tauri-bundler/src/bundle/windows/nsis/installer.nsi

```diff
@@ -656,7 +656,8 @@ Section Install
   WriteUninstaller "$INSTDIR\uninstall.exe"
 
   ; Save $INSTDIR in registry for future installations
-  WriteRegStr SHCTX "${MANUPRODUCTKEY}" "" $INSTDIR
+  WriteRegStr SHCTX "${MANUPRODUCTKEY}" "InstallDir" $INSTDIR
```

Comment:
Do we need to change this? Also I think this will break the NSIS updaters for custom install locations

Reply:
another solution is, to keep the non-value key and have the InstallDir in addition to it. This would help to make the `.msi` updater not to choose the wrong location and keep custom installations working - right?

Reply:
If we're already detecting the install location from both places in the `.msi` installer, I don't think we really need to change this for the NSIS installer?

Reply:
That is true. We can leave the `installer.nsi` changes out. It felt just more streamline having a the `InstallDir` stored in the same location in both - installer and updater. But to be fair, I don't understand how currently custom installation make use of the regex entry.

Reply:
> We can leave the `installer.nsi` changes out.

Let's do this then

> It felt just more streamline having a the `InstallDir` stored in the same location in both - installer and updater.

Yeah, I would probably want the NSIS and Wix installer saving the location to the same place, but since we have already done it differently, I don't really want to maintain 2 different locations personally, especially when it's done in a very error prone language like NSIS

Reply:
I reverted the `installer.nsi` commit and changed the comment slightly.

Let me know if it looks good for you.

Reply:
Yeah, looks good to me! Since I'm not the most familiar with the Wix installer code, I'll leave the review to other members (looks good on my part)